### DISCUSSION TITLE
What's the simplest way to get dbname-hostname pairs?

### DISCUSSION THREAD

**Discussion by rodinasophie** [cybertec-postgresql/pgwatch2#552] [Q&A]
Dear Pgwatch2 Team,

Could you please help with the following issue: I need a reliable way to group all `dbname` by their `hostname`. I don't want to reproduce the logic of `pgwatch2` where `dbname = hostname + '_' + db_name ` due to possible logic update. Are there any other reliable solutions to achieve my goal?

**Comment by pashagolub** [comment]
Hello.

Are you trying to construct SQL returning all monitored databases? 

Regards
